src/integrations/admin/first-time-configuration-integration.php

```diff
@@ -309,6 +339,22 @@ private function get_company_logo_id() {
 		return $this->options_helper->get( 'company_logo_id', '' );
 	}
 
+	/**
+	 * Gets the company logo id from the option in the database.
+	 *
+	 * @param int|string $company_logo_id The given company logo id by the user, default empty.
+	 *
+	 * @return string The company logo id.
+	 */
+	private function get_company_fallback_logo( $company_logo_id ) {
+		if ( $company_logo_id ) {
```

Comment:
A URL is passed to this function instead of an image ID.
```suggestion
	private function get_company_fallback_logo( $company_logo ) {
		if ( $company_logo ) {
```